apps/compreview-todo/todo-ssr/.npmrc

```diff
@@ -1 +1,2 @@
-registry=http://nexus-infra.apps.ocp4.example.com/repository/npm
```

Comment:
We don't need to specify the registry, because we already specify it in the command for the deployment.